scripts/auto-cpufreq-runit

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-exec /usr/local/bin/auto-cpufreq --daemon
+# test for update
+if git -c advice.detachedHead=false pull -v https://github.com/AdnanHodzic/auto-cpufreq.git | grep -q -v '^\s*\(Updating\|Already up-to-date\|From\s\)\s'; then
+    exit
+else          
+    # install the update
+    echo "i" | sudo ~/auto-cpufreq/auto-cpufreq-installer && \
+    notify-send -a "auto-cpufreq" "Update installed" && \
```

Comment:
`notify-send` doesn't work in (root) services. It shouldn't be run as root and it depends on environment variables that aren't available to runit.